src/components/DatePickerFullScreen.js

```diff
@@ -40,6 +40,16 @@ const DatePickerFullScreen = React.createClass({
     };
   },
 
+  getInitialState () {
```

Comment:
There are `defaultProps`, too. These are states that only the component handles.